src/nls/en/shortcuts-mac.json

```diff
@@ -32,7 +32,8 @@
         },
         "LAYER": {
             "CREATE_ARTBOARD": "b",
-            "DUPLICATE": "j"
+            "DUPLICATE": "j",
+            "NEW_VECTOR_MASK_FROM_SHAPE": "m"
```

Comment:
cmd M on windows is reserved for minimize. I'm happy to have one, but when i talked originally with @placegraphichere  we decided on the route of not giving windows a shortcut for now